wolfcrypt/src/aes.c

```diff
@@ -7280,6 +7304,14 @@ int  wc_AesCcmDecrypt(Aes* aes, byte* out, const byte* in, word32 inSz,
             authTagSz > AES_BLOCK_SIZE)
         return BAD_FUNC_ARG;
 
+    /* sanity check on tag size */
+    if (authTagSz != 4 && authTagSz != 6 && authTagSz != 8 &&
```

Comment:
Why isn't this check done with the new function?

Reply:
added a commit to fix it